docs/knowledgebase/runtime/storage.md

```diff
@@ -4,190 +4,100 @@ title: Runtime Storage
-
-A transparent hashing algorithm is one that makes it easy to discover and verify the input that was
-used to generate a given output. In Substrate, hashing algorithms are made transparent by
-concatenating the algorithm's input to its output. This makes it trivial for users to retrieve a
-key's original unhashed value and verify it if they'd like (by re-hashing it). The creators of
-Substrate have **deprecated the use of non-transparent hashers** within FRAME-based runtimes, so
-this information is provided primarily for completeness. In fact, it is _necessary_ to use a
-transparent hashing algorithm if you would like to access [iterable map](#iterable-storage-maps)
-capabilities. Refer to [the advanced storage documentation](../advanced/storage#storage-map-keys) to
-learn more about the important capabilities that transparent hashing algorithms expose.
+Refer to the Storage Map documentation for
+[a comprehensive list of the methods that Storage Map exposes](https://substrate.dev/rustdocs/v3.0.0/frame_support/storage/trait.StorageMap.html#required-methods).
 
-##### Common Substrate Hashers
+### Double Storage Map
 
-This table lists some common hashers used in Substrate and denotes those that are cryptographic and
-those that are transparent:
+Double storage maps are very similar to single storage maps except they contain two keys, which is useful for querying values with common keys. 
 
-| Hasher                                                                                     | Cryptographic | Transparent |
-| ------------------------------------------------------------------------------------------ | ------------- | ----------- |
-| [Blake2 128 Concat](https://substrate.dev/rustdocs/v3.0.0/frame_support/struct.Blake2_128Concat.html)   | X             | X           |
-| [TwoX 64 Concat](https://substrate.dev/rustdocs/v3.0.0/frame_support/struct.Twox64Concat.html)          |               | X           |
-| [Identity](https://substrate.dev/rustdocs/v3.0.0/frame_support/struct.Identity.html)                    |               | X           |
+Refer to the documentation on
+[advanced storage](../advanced/storage) to learn more about how different Storage Maps, including Double Storage Maps, are implemented.
 
+#### Iterating Over Storage Maps
 
-The Identity hasher encapsulates a hashing algorithm that has an output equal to its input (the
-identity function). This type of hasher should only be used when the starting key is already a
-cryptographic hash.
+Substrate Storage Maps are iterable with respect to their keys and values. Because maps are often
+used to track unbounded sets of data (such as account balances, for example), iterating over them within the runtime makes it especially likely to
+exceed block production time.
+Furthermore, because accessing the elements of a map requires more database reads than accessing the
+elements of a native list, map iterations are significantly _more_ costly than list iterations in terms of time. 
```

Comment:
```suggestion
Substrate Storage Maps are iterable with respect to their keys and values. Because maps are often
used to track unbounded sets of data (such as account balances), iterating over them without caution in the runtime may cause blocks not being able to produced in time.
Furthermore, because accessing the elements of a map requires more database reads than accessing the
elements of a native list, map iterations are significantly _more_ costly than list iterations in terms of execution time. 
```